apps/shell/test/allowlist.test.js

```diff
@@ -1,14 +1,60 @@
 /**
- * Allowlist helper function tests
+ * Helper functions tested
  */
 const helpers = require('../utils/helpers')
 
-/**
- * Constants used during test
- */
-const ALLOW_LIST = new Set(['owens.osc.edu', 'pitzer.osc.edu', '*.ten.osc.edu'])
+//generateHostAllowlist
+describe('Helper function generate_host_allowlist()', () => {
+  test('it should turn a colon-delimited string into a set', () => {
+    expect(helpers.generateHostAllowlist('pitzer.osc.edu:owens.osc.edu:*.ten.osc.edu')).toMatchObject(new Set(['pitzer.osc.edu', 'owens.osc.edu', '*.ten.osc.edu']));
+  })
+
+  test('it should return an empty set for an undefined allowlist', () => {
+    expect(helpers.generateHostAllowlist(null)).toMatchObject(new Set);
+  })
+})
+
+//pitzer expansion has v2.metadata.hidden = true so it is not included
+describe('Helper function generateClusterSshhosts()', () => {
+  test('it should include host & default values in the cluster configs', () => {
+    expect(helpers.generateClusterSshhosts('./test/clusters.d')).toMatchObject([
+      {host: 'owens.osc.edu', default: true},
+      {host: 'pitzer.osc.edu', default: false},
+      {host: 'ruby.osc.edu', default: undefined}
+    ]);
+  })
+})
+
+describe('Helper function generateDefaultSshhost()', () => {
+  test('it should return the hostname where cluster.default = true', () => {
+    const cluster_sshhosts = [{host: 'owens.osc.edu', default: true}, {host: 'pitzer.osc.edu', default: false}, {host: 'ruby.osc.edu', default: undefined}];
+    expect(helpers.generateDefaultSshhost(cluster_sshhosts)).toBe('owens.osc.edu')
+  })
+
+  test('it should return the first hostname if no cluster.default = true', () => {
+    const cluster_sshhosts_2 = [{host: 'pitzer.osc.edu', default: false}, {host: 'ruby.osc.edu', default: undefined}];
+    expect(helpers.generateDefaultSshhost(cluster_sshhosts_2)).toBe('pitzer.osc.edu')
```

Comment:
```suggestion
    const cluster_sshhosts = [{host: 'pitzer.osc.edu', default: false}, {host: 'ruby.osc.edu', default: undefined}];
    expect(helpers.generateDefaultSshhost(cluster_sshhosts)).toBe('pitzer.osc.edu')
```

You shouldn't append `_2` here, it's okay to reuse the variable name because `cluster_sshhosts` in the test above is scoped inside of its own function.